### DISCUSSION TITLE
What does the class "form-control" do?

### DISCUSSION THREAD

**Discussion by iamcarlos94** [saadeghi/daisyui#2201] [Q&A] [answered]
There is no documentation about the class "form-control". On the components' documentation (e.g. https://daisyui.com/components/radio/) it's stated that it's a container element, but what does that mean?

**Comment by YpsilonTM** [comment, marked as answer]
I think just this:
`
.form-control {
    display: flex;
    flex-direction: column;
}
`

**Reply by saadeghi** [reply]
@YpsilonTM Yes. It's helpful for having a text label and an input together.
